diff --git a/dist/testcontainers/Instance.js b/dist/testcontainers/Instance.js
index 3dc4c5fc1202ee87f35a98ff5d46a45d3eb6c3ae..941ce46fd160b7e98efb73ad50c8ab881d66b7a0 100644
--- a/dist/testcontainers/Instance.js
+++ b/dist/testcontainers/Instance.js
@@ -37,7 +37,7 @@ export const tempo = Instance.define((parameters) => {
             const promise = Promise.withResolvers();
             const containerPort = port ?? 8545;
             const c = new GenericContainer(image)
-                .withPullPolicy(PullPolicy.alwaysPull())
+                .withPullPolicy(PullPolicy.defaultPolicy())
                 .withPlatform('linux/x86_64')
                 .withExposedPorts(containerPort)
                 .withExtraHosts([
diff --git a/src/testcontainers/Instance.ts b/src/testcontainers/Instance.ts
index 90b57069efd8b893a17cdc8d738234ac5bcb4d71..fcf8c47a6a0b4d6d989cc111c8c657c84f6a31fc 100644
--- a/src/testcontainers/Instance.ts
+++ b/src/testcontainers/Instance.ts
@@ -55,7 +55,7 @@ export const tempo = Instance.define((parameters?: tempo.Parameters) => {
       const containerPort = port ?? 8545
 
       const c = new GenericContainer(image)
-        .withPullPolicy(PullPolicy.alwaysPull())
+        .withPullPolicy(PullPolicy.defaultPolicy())
         .withPlatform('linux/x86_64')
         .withExposedPorts(containerPort)
         .withExtraHosts([
