diff --git a/_cjs/tempo/chains.js b/_cjs/tempo/chains.js
new file mode 100644
index 0000000000000000000000000000000000000000..ef01d9fa93914f3c6de1081d6b82e00ae300c09d
--- /dev/null
+++ b/_cjs/tempo/chains.js
@@ -0,0 +1,11 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.tempoModerato = exports.tempoLocalnet = exports.tempoDevnet = exports.tempo = void 0;
+var tempo_js_1 = require("../chains/definitions/tempo.js");
+Object.defineProperty(exports, "tempo", { enumerable: true, get: function () { return tempo_js_1.tempo; } });
+var tempoDevnet_js_1 = require("../chains/definitions/tempoDevnet.js");
+Object.defineProperty(exports, "tempoDevnet", { enumerable: true, get: function () { return tempoDevnet_js_1.tempoDevnet; } });
+var tempoLocalnet_js_1 = require("../chains/definitions/tempoLocalnet.js");
+Object.defineProperty(exports, "tempoLocalnet", { enumerable: true, get: function () { return tempoLocalnet_js_1.tempoLocalnet; } });
+var tempoModerato_js_1 = require("../chains/definitions/tempoModerato.js");
+Object.defineProperty(exports, "tempoModerato", { enumerable: true, get: function () { return tempoModerato_js_1.tempoModerato; } });
diff --git a/_esm/tempo/chains.js b/_esm/tempo/chains.js
new file mode 100644
index 0000000000000000000000000000000000000000..a50b7557881ccf776ddd35aac84bf5b63804e459
--- /dev/null
+++ b/_esm/tempo/chains.js
@@ -0,0 +1,4 @@
+export { tempo } from '../chains/definitions/tempo.js';
+export { tempoDevnet } from '../chains/definitions/tempoDevnet.js';
+export { tempoLocalnet } from '../chains/definitions/tempoLocalnet.js';
+export { tempoModerato } from '../chains/definitions/tempoModerato.js';
diff --git a/_types/tempo/chains.d.ts b/_types/tempo/chains.d.ts
new file mode 100644
index 0000000000000000000000000000000000000000..1fd65325f7fb428037f475bbdeecff279900fc6d
--- /dev/null
+++ b/_types/tempo/chains.d.ts
@@ -0,0 +1,4 @@
+export { tempo } from '../chains/definitions/tempo.js';
+export { tempoDevnet } from '../chains/definitions/tempoDevnet.js';
+export { tempoLocalnet } from '../chains/definitions/tempoLocalnet.js';
+export { tempoModerato } from '../chains/definitions/tempoModerato.js';
diff --git a/package.json b/package.json
index 5af88cc422018d1c48c8034970b4bf10fd728bd3..5b964ba1f619ff825e59a1dffe8a5ed558dab687 100644
--- a/package.json
+++ b/package.json
@@ -119,6 +119,11 @@
       "import": "./_esm/tempo/index.js",
       "default": "./_cjs/tempo/index.js"
     },
+    "./tempo/chains": {
+      "types": "./_types/tempo/chains.d.ts",
+      "import": "./_esm/tempo/chains.js",
+      "default": "./_cjs/tempo/chains.js"
+    },
     "./tempo/actions": {
       "types": "./_types/tempo/actions/index.d.ts",
       "import": "./_esm/tempo/actions/index.js",
